doc/manual/source/installation.rst

```diff
@@ -260,6 +269,9 @@ to get started.
        
    .. group-tab:: Docker
 
+       Routinator Docker images are built with Alpine Linux for
+       ``amd64``/``x86_64`` architecture.
+       
        Due to the impracticality of complying with terms and conditions in an
        unsupervised Docker environment, before launching the container it is
```

Comment:
While this was not part of the change, I find it confusing to say "before launching the container" because the instructions to perform "before" *DO* launch the container.